script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -5,190 +5,190 @@ My body temperature drops.
 The intense cold makes it difficult just to survive.//ruby dots <た|・><だ|・><生|・><き|・><る|・>
 My brain creaks as if it's being squeezed by my skull.
 <Page2>
-Predicting the outcome of the next two minutes, I want to tear out my heart, which is already ready to burst.
+Predicting the outcome of the next two minutes, I want to tear out my heart, which is already ready to burst.//Not sure where the 'I want to' is coming from here? 俺は２分後の結末を予測し、破裂しそうな心臓をかき毟る。
 <Page3>
 Corpses swarm toward Arcueid.
 A procession reminiscent of midsummer in the midst of the extreme cold.
 They're like ants gathering sugar under the blazing sun.
 Arcueid will be killed.
-Those corpses will tear her limb from limb, just as I had, and offer her dismembered body.
+Those corpses will tear her limb from limb, just as I did, and offer up her dismembered body.//Weirdness here with the split personality 俺が／おまえが: あの死体どもに手足をもぎとられ、俺が／おまえがしたように、解体されて献上される。
 <Page4>
 All while I watch from a safe distance.
 Just like that―――#Just like that?
 <Page5>
 "―――I can't."
-That thing is a disaster in human form.
+That thing is just a disaster in human form.
```

Comment:
just is applied to the wrong word here. Vlov isn't **just** a disaster, he's a disaster that **just happens** to be in human form